page/partials/advisors.ejs

```diff
@@ -0,0 +1,81 @@
+<div class="container advisors">
+
+        <div class="row">
+            <div class="col-md-5">
+                <div class="has-carousel lazyLoad">
+                    <div>
+                        <h3>Advisors</h3>
+                        <p>"The tokenization of the equity financing could 
+                        be a major breakthrough in a capital market
+                        industry that has not seen any major change
+                        over the last 30 years (since the move from the
+                        open out cry / pit tranding to electronic
+                            trading)."</p>
+                        <div class="signature">
+                            <h3>Jean-Michel Pailhon</h3>
+                            <h6>Ledger</h6>
+                        </div>
+                    </div> <!--end advisor -->
+                    <div>
+                        <h3>Advisors</h3>
+                        <p>"The tokenization of the equity financing could 
+                        be a major breakthrough in a capital market
+                        industry that has not seen any major change
+                        over the last 30 years (since the move from the
+                        open out cry / pit tranding to electronic
+                            trading)."</p>
+                        <div class="signature">
+                            <h3>Jean-Michel Pailhon</h3>
+                            <h6>Ledger</h6>
+                        </div>
+                    </div> <!--end advisor -->
+
+
+                </div> <!-- end carousel -->
+                    
+            </div>
+            <div class="col-md-7">
+
+                <div class="row person-info-container">
```

Comment:
`person-info-container` is used two times but in different contexts. Probably you dont need it here. If you need it on line 39. If you need that margin maybe add it to `advisors` class.